File: javascript/1871.js
/*Zero vale Zero*/
let input = require("fs").readFileSync("/dev/stdin", "utf8");

let lines = input.split("\n");
let [m, n] = lines.shift().split(" ").map(Number);

do{
 let soma = m + n;
 let number = soma + "";
 let semZero = "";

for(let i of number){
  if(i !== "0"){
    semZero += i;
  }
}
 console.log(semZero);
 [m, n] = lines.shift().split(" ").map(Number);
}while(m !== 0 && n !== 0);

File: python/loop/1871.py
#1871 ZERO VALE ZERO
while True:
    valores = input().split()

    m = int(valores[0])
    n = int(valores[1])

    if m == 0 and n == 0:
        break

    soma = m + n
    sem_zero = str(soma).replace('0', '')
    print(sem_zero)